### ISSUE TITLE
Not the same package is installed 

### ISSUE THREAD

**Issue by mathieugravil** [Azure/azure-sdk-for-python#9235]
Hello,

I am writing some package based on azure sdk. So In my setup.py, i put :
 install_requires=[
...
'azure-keyvault-secrets==4.0.0'
..]
When i install my package it install 'azure-keyvault-secrets==4.0.0' but not in the same way than if i do pip install 'azure-keyvault-secrets==4.0.0'

And i get some error : 

Traceback (most recent call last):
  File "tests/test_common.py", line 4, in <module>
    from azure.keyvault.secrets import SecretClient
ImportError: No module named keyvault.secrets

If i install package with pip command i have no error.

So I have compared pacakge installed with setup.py and with pip installation:
with pip installation : 

pip show gives me : 
Name: azure-keyvault-secrets
Version: 4.0.0
Summary: Microsoft Azure Key Vault Secrets Client Library for Python
Home-page: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-secrets
Author: Microsoft Corporation
Author-email: azurekeyvault@microsoft.com
License: MIT License
Location: /usr/local/lib/python2.7/site-packages
Requires: azure-core, enum34, typing, msrest, azure-keyvault-nspkg, azure-common
Required-by: 

and  with setup:

Name: azure-keyvault-secrets
Version: 4.0.0
Summary: Microsoft Azure Key Vault Secrets Client Library for Python
Home-page: https://github.com/Azure/azure-sdk-for-python/tree/master/sdk/keyvault/azure-keyvault-secrets
Author: Microsoft Corporation
Author-email: azurekeyvault@microsoft.com
License: MIT License
Location: /usr/local/lib/python2.7/site-packages/azure_keyvault_secrets-4.0.0-py2.7.egg
Requires: azure-common, azure-core, azure-keyvault-nspkg, enum34, msrest, typing
Required-by: az-k8s-operations

So location is not the same and if i have a look on files : 
setup.py : 

# ls -lart /usr/local/lib/python2.7/site-packages/azure_keyvault_secrets-4.0.0-py2.7.egg
total 28
drwxr-sr-x 3 root staff  4096 Dec 21 07:39 azure
drwxr-sr-x 4 root staff  4096 Dec 21 07:39 .
drwxr-sr-x 2 root staff  4096 Dec 21 07:39 EGG-INFO
drwxrwsr-x 1 root staff 12288 Dec 21 07:39 ..

pip installation : 

# ls -lart /usr/local/lib/python2.7/site-packages/azure_keyvault_secrets-4.0.0.dist-info/
total 68
-rw-r--r-- 1 root staff     6 Dec 21 07:55 top_level.txt
-rw-r--r-- 1 root staff   110 Dec 21 07:55 WHEEL
-rw-r--r-- 1 root staff 22770 Dec 21 07:55 METADATA
drwxrwsr-x 1 root staff 12288 Dec 21 07:55 ..
-rw-r--r-- 1 root staff  9010 Dec 21 07:55 RECORD
-rw-r--r-- 1 root staff     4 Dec 21 07:55 INSTALLER
drwxr-sr-x 2 root staff  4096 Dec 21 07:55 .

So It seems me strange. It should be the same...

**Comment by lmazuel** [comment]
Hi @mathieugravil 
Could you clarify:
- The version of pip used
- The version of setuptools installed in your env
- The command you type to install your own setup.py with the install_requires. Ideally, the full log of that command would be awesome.
- Do you have a file `__init__.py` file in /usr/local/lib/python2.7/site-packages/azure_keyvault_secrets-4.0.0-py2.7.egg/azure and if yes, what is inside?

**Comment by lmazuel** [comment]
Closing for inactivity, if the problem persists, please open a new issue with all the details asked in the previous post. Thanks!
